<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestSuite</class>
 <widget class="QMainWindow" name="TestSuite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>640</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>TestSuite</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/applications_games.png</normaloff>images/applications_games.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5"/>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Test Collection</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QListWidget" name="listTestCollection">
             <property name="showDropIndicator" stdset="0">
              <bool>true</bool>
             </property>
             <property name="dragEnabled">
              <bool>false</bool>
             </property>
             <property name="dragDropOverwriteMode">
              <bool>false</bool>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::NoDragDrop</enum>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPushButton" name="buttonAdd">
               <property name="toolTip">
                <string>Add test case</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>images/add.png</normaloff>images/add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Edit test case</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>images/edit.png</normaloff>images/edit.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonRemove">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Remove test case</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>images/remove.png</normaloff>images/remove.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonRemoveAll">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Remove all test cases</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>images/remove_all.png</normaloff>images/remove_all.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2" rowspan="3">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Log</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="buttonClearLog">
         <property name="toolTip">
          <string>Clear log</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/clear_log.png</normaloff>images/clear_log.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>232</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTextBrowser" name="textBrowserLog"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>90</height>
       </size>
      </property>
      <property name="title">
       <string>Execution</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="buttonStop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="buttonExecute">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Execute</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="buttonExecuteAll">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Execute All</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="buttonExecuteFailed">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Execute Failed</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="buttonExecuteUntested">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Execute Untested</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Execution Speed:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>0 ms</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderSpeed">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>200</number>
           </property>
           <property name="pageStep">
            <number>200</number>
           </property>
           <property name="value">
            <number>200</number>
           </property>
           <property name="sliderPosition">
            <number>200</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>200</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>1000 ms</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>123</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBoxStopWhenFailed">
         <property name="text">
          <string>Stop when test case fails</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBoxStopOnError">
         <property name="text">
          <string>Stop on error</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>180</height>
       </size>
      </property>
      <property name="title">
       <string>Test Case Info</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Hand:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelHand">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelStatus">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Info:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTextBrowser" name="textInfo">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
